docs/source/setup/presentation.rst

```diff
@@ -112,21 +110,21 @@ Process Tab
 Click the :code:`Process` tab on the left side of the window.
 
 .. image:: ../_images/setup/presentation/process_tab_location.png
-   :alt: Arrow pointing out the process tab location
+   :alt: Arrow pointing out the process tab location.
    :align: center
 
 You should now see four options in the center of the screen:
```

Comment:
```suggestion
You should now see many options in the center of the screen:
```

since the moment I created this page the number of available recipe has grown a bit ;-)